src/kartikeyDJune20/FirstLastIndexInString.java

```diff
@@ -0,0 +1,24 @@
+package kartikeyDJune20;
+
+public class FirstLastIndexInString {
+	
+	static void firstLastIndexInString(String input) {
+		
+		for(int index=0;index< input.length();index++) {
+			int firstIndex=0;
+			int lastIndex=0;
+			char ch=input.charAt(index);
+			firstIndex=input.indexOf(ch);
+			lastIndex=input.lastIndexOf(ch);
+			if(firstIndex!=index && lastIndex!=0)
```

Comment:
Are you sure you need so many local variable for this, no!!!
for (int index = 0; index < name.length(); index++) {
			char ch = name.charAt(index);
			if (index == name.indexOf(ch) && name.indexOf(ch) < name.lastIndexOf(ch)) 
					System.out.println(
							ch + " -> first index = " + name.indexOf(ch) + " Last index = " + name.lastIndexOf(ch));
		}